lib/git-version-bump.rb

```diff
@@ -249,9 +249,14 @@ def self.caller_file
 		# within this file, we can't just look at Gem.location_of_caller, but
 		# instead we need to parse the caller stack ourselves to find which
 		# gem we're trying to version all over.
+		#
+		# Caller returns paths of the form `/path/to/file:100:in method...` on *nix
+		# and `C:\path\to\file:in method...` on Windows. Splitting on colon isn't
+		# reliable since the Windows path has a colon at the beginning, split on :num:
+		# instead since that will always be there no matter the platform.
 		Pathname(
 		  caller.
-		  map  { |l| l.split(':')[0] }.
+		  map  { |l| l.split(/:\d+:/)[0] }.
```

Comment:
It's probably time that we declare Ruby 1.x dead and buried, and switch to using `caller_locations`.  That'll avoid all need for text manipulation.

Reply:
Yep that was much easier, fixed.